### ISSUE TITLE
vue map import

### ISSUE THREAD

**Issue by rcrath** [felixhayashi/TW5-TiddlyMap#61]
related to request for rdf import, https://github.com/felixhayashi/TW5-TiddlyMap/issues/60.  VUE (http://vue.tufts.edu) is an open source program written in java that draws maps similar to tiddlymap.  development is currently dormant, but there are a lot of users, including me, who would be interested in tiddlymap, especially if existing vue maps could be imported.  The map files are XML files, so should be conducive to conversion I would think (though I haven't a clue as to how!).  Thanks for considering, and for developing this.

**Comment by felixhayashi** [comment]
> especially if existing vue maps could be imported.

You have a point here. As said in #60, Import export of graphs should be somewhat possible. A similar issue is #33.

I am about to improve the way tiddlymap persists the graphs (nodes/edges) and when I am done and I am confident with the data structure, I will publish the api so plugin authors will get a chance to write their own parsers for importing exporting graphs. Then I do not need to program import/export functionality for different formats myself... Anyways, I keep this issue open and in the back of my head.

-Felix

**Comment by rcrath** [comment]
thanks.  not able to code much myself beyond css/html stuff

**Comment by felixhayashi** [comment]
The vue format does not seem to hard to parse. I will start writing import modules at some point but it won't be in the near future I am afraid :(
